src/docs/product/dashboards/index.mdx

```diff
@@ -16,13 +16,22 @@ If you’re interested in being an Early Adopter, you can turn your organization
 
 </Note>
 
-Dashboards provide the ability to build custom views of your application's health, including the ability to navigate issues and performance across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/).
+Dashboards will enable you to get a broad overview of your application’s health. You’ll be able to navigate through error and performance data across multiple projects and zoom in on any time series visualizations you may want to investigate. All widgets in the same view will update synchronously to reflect the new date range in the Global Selection Header.
```

Comment:
Rewording for brevity. Good rule of thumb to replace "will [verb]" with one verb wherever you can unless the thing you're talking about is truly going to only happen in the future. 

The last sentence talks about widgets without having introduced them, so I edited this so that widgets get introduced. It's closer to the original text but without mentioning building custom dashboards. Also moved some text to after the image.

```suggestion
Dashboards provide you with a broad overview of your application’s health by allowing you to navigate through error and performance data across multiple projects. Dashboards are made up of one or more widgets, and each widget visualizes one or more [Discover queries](/product/discover-queries/). 
```

Reply:
Changed